packages/registry/package.json

```diff
@@ -71,20 +94,26 @@
     "crypto-random-string": "3.2.0",
     "debug": "4.3.2",
     "dependency-graph": "0.10.0",
-    "glob": "7.1.6"
+    "glob": "^10.4.5",
+    "tmp": "0.2.1"
   },
   "devDependencies": {
     "@parcel/packager-ts": "^2.12.0",
     "@parcel/transformer-typescript-types": "^2.12.0",
     "@plone/types": "workspace:*",
+    "@types/debug": "^4.1.12",
```

Comment:
Is this really a needed dependency? I haven't seen it used, but it's also a lot of changes so I might have missed it.

Reply:
yeah, if not, TS complains:

https://github.com/plone/volto/pull/6399/files#diff-ddb86bc31c1cc3321384d0b8e088fba1fe6233801cc9e7e6ff2e5c368e9f67f2R5